seshdash/static/sesh-ui/js/sesh_dashboard.js

```diff
@@ -239,8 +237,68 @@ get_high_chart( date, HighChartHighPvProduction, HighChartHighCloudCover);
                modal.modal('hide');
           });
       });
-     
 
-     
                   }
-      
+
+      /* Auto compete on search*/
+var i;
+var textid;
+var textinput;
+var option;
+var matched;
+var siteid = [];
+var sitename = [];
+$.post("/search",{csrfmiddlewaretoken: csrftoken},function(data){
+    option = JSON.parse(data);
+   for (i=0;i < option.length;i++){
+    sitename.push(option[i].value)
+   }
+   for (i=0; i< option.length; i++){
+    siteid.push(option[i].key)
+   }
+   var input = document.getElementById('search');
+   new Awesomplete(input,{list:sitename});
+});
+
```

Comment:
@Frankkwizera  Please add some comments in this JS
It's very difficult to read as it is.